test/e2e/autoscaling-max-tasks-per-minute/actor/main.js

```diff
@@ -1,12 +1,18 @@
 import { Actor } from 'apify';
 import { BasicCrawler, log as defaultLog, LogLevel } from '@crawlee/basic';
+import { ApifyStorageLocal } from '@apify/storage-local';
 
 const crawlerLogger = defaultLog.child({
     prefix: 'AutoscalingTest',
     // Set this to false if you want to see verbose output from the autoscaled pool
     level: true ? defaultLog.getOptions().level : LogLevel.PERF,
 });
 
+const mainOptions = {
```

Comment:
Should we maybe make this a common util function instead of duplicating the object definition? Can we even do that?

Reply:
Probably not, as this `actor` folder is pushed to the platform as-is, thus we don't push files out of it to the platform -> it will just fail there as it would not know from where it should import this function...

Reply:
Ah, right, forgot about that, sorry! Still, looks fine to me 👍